RHEL CI needs btrfs-tools or btrfs-progs-devel

It seems that RHEL CI needs btrfs-tools or btrfs-progs-devel in order to build for example CRI-O
```
13:30:53 # github.com/containers/storage/drivers/btrfs
13:30:53 vendor/github.com/containers/storage/drivers/btrfs/btrfs.go:8:10: fatal error: btrfs/ioctl.h: No such file or directory
13:30:53  #include <btrfs/ioctl.h>
13:30:53           ^~~~~~~~~~~~~~~
13:30:53 compilation terminated.
13:31:45 make: *** [Makefile:141: bin/crio] Error 2
13:31:45 Failed at 92: make BUILDTAGS='exclude_graphdriver_devicemapper libdm_no_deferred_remove'
13:31:45 CRI-O not installed
```

--- a/.ci/install_crio.sh
+++ b/.ci/install_crio.sh
@@ -89,7 +89,7 @@ fi
 pushd "${GOPATH}/src/${crio_repo}"
 echo "Installing CRI-O"
 make clean
-make BUILDTAGS='exclude_graphdriver_devicemapper libdm_no_deferred_remove'
+make BUILDTAGS='exclude_graphdriver_btrfs exclude_graphdriver_devicemapper libdm_no_deferred_remove'
 make test-binaries
 sudo -E PATH=$PATH sh -c "make install"
 sudo -E PATH=$PATH sh -c "crio config --default > crio.conf"

